recipes/pcrunch/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "pCrunch" %}
+{% set version = "1.0.3" %}
+{% set release = "v1.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/nrel/pCrunch/archive/{{ release }}.tar.gz
```

Unless you think this is going to be more different, you could just do:
```suggestion
{% set version = "1.0.3" %}

package:
  name: {{ name|lower }}
  version: {{ version }}

source:
  url: https://github.com/nrel/pCrunch/archive/v{{ version }}.tar.gz
```

---

simplified as suggested